app/src/main/java/com/github/fribourgsdp/radio/StringComparisons.kt

```diff
@@ -0,0 +1,98 @@
+package com.github.fribourgsdp.radio
+
+import java.text.Normalizer
+
+const val MAX_ALLOWED_ERRORS = 2
+
+class StringComparisons {
+    companion object{
+        private const val chars = "abcdefghijklmnopqrstuvwxyz1234567890"
+
+        /**
+        * Function that returns a number of points depending on the number of mistakes in the comparison of the two provided strings.
+        */
+        fun compareAndGetPoints(actual : String, expected : String, maxPts : Int = 100) : Int {
+            val oneErrorPenalty = 0.9
+            val twoErrorsPenalty = 0.8
+            return when(compare(actual, expected)){
+                0 -> maxPts
+                1 -> (oneErrorPenalty*maxPts).toInt()
+                2 -> (twoErrorsPenalty*maxPts).toInt()
+                else -> 0
+            }
+        }
+
+        /**
+        * Function that compares two strings and returns the number of differences between them.
+        * Returns -1 if the number of differences is bigger than 2, to avoid taking too much time.
+        */
```

Comment:
Don't you think we could also return a value from an enum for clarity?

Reply:
Maybe not an enum, because this method is made to return integers (the maximum number can be adjusted with a constant). But it can be good to return a constant.